components/icon/Icon.js

```diff
@@ -30,7 +30,7 @@ Icon.propTypes = {
 
 Icon.defaultProps = {
     viewBox: '0 0 16 16',
-    className: 'icon-16p fill-global-grey'
```

Comment:
Are you sure `fill-global-grey` is not anymore needed?

Reply:
cf screenshot

Reply:
TBH, I'm not sure this will work on all devices (should be more tested, I don't know how browsers will behave in this case).
By default, this is black, not using Proton's global grey.

Reply:
`Dans l'absolu, on ne devrait jamais l'instancier sans une couleur ces icônes, sauf en cas de composant particulier comme le toggle`

So default grey and for Toggle we use none